libcxx/include/__algorithm/ranges_minmax.h

```diff
@@ -83,7 +85,18 @@ struct __fn {
 
     _LIBCPP_ASSERT_VALID_ELEMENT_ACCESS(__first != __last, "range has to contain at least one element");
 
-    if constexpr (forward_range<_Range>) {
+    if constexpr (contiguous_range<_Range> && is_integral_v<_ValueT> &&
```

Comment:
I think we should look into optimizing `minmax_element` instead for the same conditions. If that doesn't work, it may be acceptable to do this here instead. A comment would be good.